Sources/apollo-ios-pagination/UpdateSource.swift

```diff
@@ -0,0 +1,3 @@
+public enum UpdateSource {
+  case fetch, cache
+}
```

Comment:
This is functionally equivalent to `GraphQLResult<Data>.Source`, but without needing a specified `Data`. A source is a source is a source; and needing to specify a specific `Data` here complicates usage without conferring much benefit.